wondrous-bot-admin/src/pages/quests/QuestResultsPage.tsx

```diff
@@ -89,6 +117,30 @@ const QuestResultsPage = () => {
         isEditMode,
       }}
     >
+      <Modal
```

Comment:
should i somehow refactor this? actually not sure how

Reply:
No need for now - unless there's more complicated modal logic